src/components/PaymentTile.js

```diff
@@ -85,6 +88,13 @@ const PaymentTile = (props) => {
     const handleEditPayment = () => {
         history.push(`/clients/${client.id}/payments/${payment.id}/update`)
     }
+    const handleRedirect = () => {
+        history.push(`/clients/${client.id}`)
+        localStorage.setItem('openPayment', payment.id)
```

Comment:
Instead of `localStorage` we should just use the URL params so you would have

```
history.push(`/clients/${client.id}?paymentId=${payment.id}`)
```

And then you can just pull it out of the URL in [PaymentsList.js](https://github.com/setlife-network/trinary/pull/674/files#diff-c1738c305cf76b78901c816f6b869a371c366ddd6ecdcfa20f669806574bc7b3R25)